google_guest_agent/protos/snapshot_service.proto

```diff
@@ -0,0 +1,99 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
do we need to check the proto file in here..?

Reply:
i don't have any context for doing this, it seems harmless but also useless. all of our other proto work relies on published protos